.github/workflows/prepare_release.yml

```diff
@@ -0,0 +1,130 @@
+name: prepare_release
+
+on:
+  push:
+    branches: ['prepare-release-workflow']
```

Comment:
This is only for demonstration purposes with this PR. Once it merges, we will run this workflow manually from the Actions tab of our repo.